chromium_src/net/base/lookup_string_in_fixed_set.cc

```diff
@@ -32,23 +32,23 @@ int LookupSuffixInReversedSet(base::span<const uint8_t> graph,
   // omnibox, it will be parsed as OmniboxInputType::URL input type instead of
   // OmniboxInputType::UNKNOWN, The first entry in the autocomplete list will be
   // URL instead of search.
-  for (auto* unstoppable_domain : decentralized_dns::kUnstoppableDomains) {
-    if (host.ends_with(unstoppable_domain)) {
-      *suffix_length = strlen(unstoppable_domain) - 1;
-      return kDafsaFound;
-    }
+  if (const auto found = decentralized_dns::kUnstoppableDomains.find(
+          decentralized_dns::DomainSuffixLookup(host));
+      found != decentralized_dns::kUnstoppableDomains.end()) {
+    *suffix_length = found->size();
+    return kDafsaFound;
   }
   if (host.ends_with(decentralized_dns::kEthDomain)) {
-    *suffix_length = strlen(decentralized_dns::kEthDomain) - 1;
+    *suffix_length = decentralized_dns::kEthDomain.size();
```

Comment:
I don't quite get why `- 1` was previously there, but now we just use `size()`. Was the `- 1` a mistake before?

Reply:
Fixed. The `-1` is for the `.` in the constant (I think).